File: Rajwansh_java_training/session1/controlflow/MultiplicationTable.java
package session1.controlflow;

import java.util.Scanner;

public class MultiplicationTable {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter number: ");
        int n = sc.nextInt();

        System.out.println("Multiplication Table of " + n);

        //  Here I used a for loop to iterate from 1 to 10 and print the multiplication table of the given number n. The loop runs 10 times, multiplying n by the loop variable i and printing the result in a formatted way.
        for (int i = 1; i <= 10; i++) {
            System.out.println(n + " x " + i + " = " + (n * i));
        }

        sc.close();
    }
}

File: Rajwansh_python_training/python_basic/loops/multiplication_table.py
TABLE_LIMIT = 10


def print_multiplication_table(number: int) -> None:
    """
    Print multiplication table for the given number.
    """

    # Run the loop from 1 to TABLE_LIMIT.
    for multiplier in range(1, TABLE_LIMIT + 1):
        result = number * multiplier

        print(f"{number} x {multiplier} = {result}")


def main() -> None:
    """Take input and display multiplication table."""

    user_number = int(input("Enter a number: "))

    print_multiplication_table(user_number)


if __name__ == "__main__":
    main()
